modules/scripts/startup-script/files/setup-raid.yml

```diff
@@ -0,0 +1,88 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+
+- name: Configure local SSDs
+  become: true
+  hosts: localhost
+  vars:
+    DST_MNT: "/mnt/localssd"
+    bashrc: "{{ '/etc/bashrc' if ansible_facts['os_family'] == 'RedHat' else '/etc/bash.bashrc' }}"
+    array_dev: "/dev/md0"
+    mode: '0755'
+  tasks:
+  - name: Check if {{ array_dev }} exists
+    ansible.builtin.stat:
+      path: "{{ array_dev }}"
+    register: array_dev_exists
+
+  - name: Install mdadm
+    ansible.builtin.package:
+      name: mdadm
+      state: present
+
+  - name: Get number of devices
```

Comment:
Folllowing our documentation for [predictable naming of local SSD devices](https://cloud.google.com/compute/docs/disks/disk-symlinks
), I suggest replacing these 2 tasks with:

```yaml
           vars:
              interface: nvme
...
            - name: Get local SSD devices
              ansible.builtin.find:
                file_type: link
                path: /dev/disk/by-id
                patterns: google-local-{{ "nvme-" if interface == "nvme" else "" }}ssd-*
              register: local_ssd_devices
```

Reply:
Done.